app/tests/unit/routes/v1/user.spec.js

```diff
@@ -0,0 +1,97 @@
+const request = require('supertest');
+
+const { expressHelper } = require('../../../common/helper');
+const validator = require('../../../../src/validator/user');
+
+//
+// Mock middleware, we are not testing this here
+//
+jest.mock('../../../../src/middleware/authorization', () => ({
+  checkAppMode: jest.fn((_req, _res, next) => next()),
+}));
+
+jest.mock('../../../../src/middleware/featureToggle', () => ({
+  requireDb: jest.fn((_req, _res, next) => next()),
+  requireSomeAuth: jest.fn((_req, _res, next) => next()),
+}));
+
+jest.mock('../../../../src/validator/user');
+
+//
+// Mocks are in place, create the router
+//
+const router = require('../../../../src/routes/v1/user');
+
+const { userController } = require('../../../../src/controllers');
+
+// Express Server
+const basePath = '/api/v1/user';
+const app = expressHelper(basePath, router);
+
+// Mock config library - @see {@link https://stackoverflow.com/a/64819698}
+jest.mock('config');
+
+describe(`GET ${basePath}`, () => {
+  // TODO: Ensure other middleware funcs are called once
+  const validatorSearchUserSpy = jest.spyOn(validator, 'searchUsers');
+  const ctrlrSearchUserSpy = jest.spyOn(userController, 'searchUsers');
+
+  beforeEach(() => {
+    validatorSearchUserSpy.mockReset();
+    ctrlrSearchUserSpy.mockReset();
```

Comment:
We may be able to avoid needing to explicitly spy reset declare on all of these if we use `jest.resetAllMocks()` at a higher beforeEach level?
Ref: <https://jestjs.io/docs/jest-object#jestresetallmocks>